Added "curated dashboards" information and broke down, rearranged topics. (#30659)
grafana/grafana: docs/sources/datasources/google-cloud-monitoring/preconfig-cloud-monitoring-dashboards.md

+++

> title = "Preconfigred dashboards"

Reviewer: ```suggestion
title = "Preconfigured dashboards"
```

Author: Done

description = "Guide for using Google Cloud Monitoring in Grafana"
keywords = ["grafana", "stackdriver", "google", "guide", "cloud", "monitoring"]
aliases = ["/docs/grafana/latest/features/datasources/stackdriver", "/docs/grafana/latest/features/datasources/cloudmonitoring/"]
weight = 10
+++

# Preconfigured Cloud Monitoring dashboards

Google Cloud Monitoring data source ships with pre-configured dashboards for some of the most popular GCP services. These curated dashboards are based on similar dashboards in the GCP dashboard samples repository. See also, [Using Google Cloud Monitoring in Grafana]({{< relref "./_index.md" >}}) for detailed instructions on how to add and configure the Google Cloud Monitoring data source.
## Curated dashboards

The curated dashboards are based on similar dashboards in the GCP dashboard samples repository.

To import the curated dashboards:

> 1. Go to the configuration page of your Cloud Monitoring data source and click on the `Dashboards` tab.

Reviewer: ```suggestion
1. On the configuration page of your Cloud Monitoring data source, click the **Dashboards** tab.
```

> 2. Click `Import` for the dashboard you would like to use.

Reviewer: ```suggestion
1. Click **Import** for the dashboard you would like to use.
```

+++
title = "Preconfigred dashboards"
description = "Guide for using Google Cloud Monitoring in Grafana"
keywords = ["grafana", "stackdriver", "google", "guide", "cloud", "monitoring"]
aliases = ["/docs/grafana/latest/features/datasources/stackdriver", "/docs/grafana/latest/features/datasources/cloudmonitoring/"]
weight = 10
+++

# Preconfigured dashboards

Google Cloud Monitoring data source ships with [out-of-the-box](#out-of-the-box-dashboards) and [curated](#curated-dashboards) dashboards. See also [Using Google Cloud Monitoring in Grafana]({{< relref "./_index.md" >}}) for detailed instructions on how to add and configure the data source.

## Out-of-the-box dashboards

Grafana v7.3 and later versions supports five of the most popular out-of-the-box GCP services. They are:

> - BigQuery

Reviewer 2: Sorry Jita, I didn't know we already have the ## Out-of-the-box dashboards section. Out-of-the-box dashboards are the same thing as curated dashboards, so we need to remove the whole ##Out-of-the-box section.

Author: @sunker, thank you for the review. The original topic was so long, it was easy to overlook existing content. Let me know if you have any other comments/suggestions.

- Cloud Load Balancing
- Cloud SQL
- Google Compute Engine `GCE`
- Google Kubernetes Engine `GKE`

To import the out-of-the-box dashboards:

1. Go to the configuration page of a Cloud monitoring data source and click on the `Dashboards` tab.

2. Click `Import` for the dashboard you would like to use.

> The data source of the newly created dashboard panels will be the one selected above. The dashboards have a template variable that is populated with the projects accessible by the configured service account every time the dashboard is loaded. After the dashboard is loaded, you can select the project you prefer from the drop-down list.

Reviewer 2: This paragraph could be moved to the ## Curated dashboards section (after Click `Import` for the dashboard you would like to use)

In case you want to customize a dashboard, we recommend that you save it under a different name. Otherwise the dashboard will be overwritten when a new version of the dashboard is released.

{{< docs-imagebox img="/img/docs/v73/cloud-monitoring-dashboard-import.png" max-width= "700px" caption="Cloud Monitoring dashboard import" >}}

## Curated dashboards

> Grafana v7.4 and later versions support some of the most popular GCP services. These curated dashboards are based on similar dashboards in the GCP dashboard samples repository.

Reviewer 2: I'm not sure about the word "support" here. It's not like this wasn't supported before, it's just that now they are pre-configured when you install the datasource. 

```suggestion
The Cloud Monitoring data source ships with pre-configured dashboards for some of the most popular GCP services. These curated dashboards are based on similar dashboards in the GCP dashboard samples repository.
```
